02_configure_host.sh

```diff
@@ -176,7 +176,14 @@ for IMAGE_VAR in $(env | grep "_LOCAL_IMAGE=" | grep -o "^[^=]*") ; do
   export $IMAGE_VAR="${IMAGE##*/}"
   #shellcheck disable=SC2086
   export $IMAGE_VAR="${REGISTRY}/localimages/${!IMAGE_VAR}"
-  sudo "${CONTAINER_RUNTIME}" build -t "${!IMAGE_VAR}" . -f ./Dockerfile
+
+  # Support building ironic-image from source
+  if [[ "${IMAGE}" =~ "ironic-image" ]] && [[ ${IRONIC_FROM_SOURCE:-} == True ]]; then
```

Comment:
nit: just to make easier since it is key sensitive 
```suggestion
  if [[ "${IMAGE}" =~ "ironic-image" ]] && [[ ${IRONIC_FROM_SOURCE:-} == true ]]; then
```

Reply:
I think we use True in other places?

Reply:
it doesn't really matter, it makes sense to change it